Foundation/FileManager.swift

```diff
@@ -549,6 +646,31 @@ open class FileManager : NSObject {
         This method replaces createDirectoryAtPath:attributes:
      */
     open func createDirectory(atPath path: String, withIntermediateDirectories createIntermediates: Bool, attributes: [FileAttributeKey : Any]? = [:]) throws {
+#if os(Windows)
+        var saAttributes: SECURITY_ATTRIBUTES =
+            SECURITY_ATTRIBUTES(nLength: DWORD(MemoryLayout<SECURITY_ATTRIBUTES>.size),
+                                lpSecurityDescriptor: nil,
+                                bInheritHandle: FALSE)
+        let psaAttributes: UnsafeMutablePointer<SECURITY_ATTRIBUTES> =
+            UnsafeMutablePointer<SECURITY_ATTRIBUTES>(&saAttributes)
+
+        try path.withCString(encodedAs: UTF16.self) {
```

Comment:
I think that this is broken as there is a callback that needs to occur on each intermediate creation which this does not do.